### PR TITLE
Update how-should-i-integrate.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Wording fix.

### CODE REVIEW

**Review by hanzei — APPROVED** [review]

File: site/content/integrate/getting-started/how-should-i-integrate.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ To build a richer bot integration, you can make full use of the [Mattermost REST
 
 ### I want to build an interactive app
 
-That's no problem. Our Apps provide a framework with utilities like a KV store, bot account creation, OAuth permission scopes, and rich UX entry points.
+That's no problem. Our Apps provide a framework with utilities like a key-value store, bot account creation, OAuth permission scopes, and rich UX entry points.
```
Comment (mickmister) [inline site/content/integrate/getting-started/how-should-i-integrate.md:31]: Honestly the OAuth permission scopes are pretty limited at the moment, so I don't know if we want to advertise them. Thoughts on this @hanzei?

Comment (hanzei) [reply site/content/integrate/getting-started/how-should-i-integrate.md:31]: They are one selling point which we will improve in the future. 1/5 to keep them in.

Comment (justinegeffen) [reply site/content/integrate/getting-started/how-should-i-integrate.md:31]: It depends on the speed at which we're iterating tbh. Might be helpful to leave it in?

**Comment by justinegeffen** [comment]
@mickmister, are you good to approve and merge?

**Review by mickmister — APPROVED** [review]

**Pull request merged**